CMakeLists.txt

```
@@ -286,6 +294,15 @@ if(ENABLE_TESTING OR ENABLE_PROGRAMS)
         PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/tests/include
         PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/include
         PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/library)
+    # Pass-through MBEDTLS_CONFIG_FILE and MBEDTLS_USER_CONFIG_FILE
+    if(DEFINED MBEDTLS_CONFIG_FILE)
```

```suggestion
    if(MBEDTLS_CONFIG_FILE)
```
or
```suggestion
    if(EXISTS MBEDTLS_CONFIG_FILE)
```

See my comments above.